core/posts/serializers.py

```diff
@@ -4,7 +4,7 @@
 from core.authors.serializers import get_summary
 from core.hostUtil import get_host_url
 
-PAGE_SIZE = 50
+PAGE_SIZE = 5
```

Comment:
Not sure, but https://github.com/i-5ive/CMPUT404-project-socialdistribution/blob/9aff5c342ee85c223f368778b1aa2cf3a415c6b7/example-article.json#L77, has the size as 50.

Reply:
Idk either, but the lines below it say to choose a comment size like 5 so yea

Reply:
Yeah, we'll know sooner or later... this should be fine